apps/docs/security/github-scanning.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: "GitHub Secret Scanning"
+description: "Learn about"
```

Comment:
This shows up in the og image too and adds no value.
"How Unkey protects you from leaked root keys" or similar would be better imo